README.md

```diff
@@ -35,3 +35,21 @@ require('babel-core').transform('code', {
   plugins: ['dynamic-import-node']
 });
 ```
+
+### Code Example
+```javascript
+Promise.all([
+      /* jshint ignore:start */
```

Comment:
i'm not sure why we need override comments from an obsolete linter in our code examples; separately, our code examples should be compliant with the airbnb style guide (2-space indentation, for one)